source/initial_temperature/patch_on_S40RTS.cc

```diff
@@ -39,7 +39,7 @@ namespace aspect
     void
     PatchOnS40RTS<dim>::initialize ()
     {
-      Utilities::AsciiDataInitial<dim>::initialize(1);
+      this->Utilities::AsciiDataInitial<dim>::initialize(1);
```

Comment:
Is this necessary, or an accidental change?

Reply:
Wolfgang preferred this notation but its not strictly necessary - sorry, it maybe should have been a separate PR.